### DISCUSSION TITLE
Levels 1/2/3: Which new hardware should go to which?

### DISCUSSION THREAD

**Discussion by erkinalp** [EtchedPixels/FUZIX#1052] [Ideas]
We currently have certain hardware that is defined as level 1 and only includes essential components; and hardware that is defined as level 2 and including all relevant utilities.
Level 3 would be a logical expansion over level 2 with a simple GUI and expanded file systems and/or complex boot methods compatibility.
This discussion thread is started to enjoy and resolve the discussions about the "level" of a particular hardware target.

**Comment by EtchedPixels** [comment]
No and I have no plans to change it.

Level 0 is a minimal setup, which requires no special access methods needed for user/kernel data, and no verification of address ranges. 
Level 1 is for 8bit systems
Level 2 enables certain later Unix features that don't scale well on a small device: session management, rlimit, group lists so is applicable to 16/32bit systems.
